### ISSUE TITLE
Optional layer of VPAT conformance ratings

### ISSUE THREAD

**Issue by slhenry** [w3c/wai-atag-report-tool#155]
I came across this issue for the WCAG-EM Report Tool (from @mitchellevan ):
**[Add an optional layer of VPAT conformance ratings](https://github.com/w3c/wcag-em-report-tool/issues/344)**

I assume it applies for this ATAG Report Tool as well. And I assume we'd not do it with Release 1, and label it for later iteration &mdash; especially since it's Priority 3 for WCAG-EM Report Tool. I'll leave that for Hidde and Shadi to work out. :-)

(and maybe there are some other issues on WCAG-EM Report Tool that apply also to this ATAG Report Tool?)

**Comment by mitchellevan** [comment]
I agree this is an opportunity for ATAG too. However, a ratings layer alone is not enough for ATAG-to-VPAT because the clauses do not map one-to-one. For example:

* ATAG A1.1.1 maps to a set of "web" responses for WCAG 2.0 clauses in VPAT (one-to-many mapping)
* ATAG B.2.1.1 maps to a set of "authoring tool" responses for WCAG 2.0 clauses in VPAT (one-to-many mapping)
* ATAG B.1.2.1 Restructuring and Recoding Transformations (WCAG) maps fairly closely to specific non-WCAG requirements in the international VPAT, such as:
** Section 508 clause 504.2.1 Preservation of Information Provided for Accessibility in Format Conversion
** EN 301 549 clause 11.6.3 Preservation of accessibility information in transformations
* ATAG A.3.7.1 Preview (Minimum) does not clearly map to specific requirements in the international VPAT.
